pkg/kubernetes/client.go

```diff
@@ -157,3 +157,13 @@ func (c Client) ListServices() []*corev1.Service {
 func (c Client) GetAnnouncementsChannel(informerID InformerKey) <-chan interface{} {
 	return c.announcements[informerID]
 }
+
+// GetNamespace returns a Namespace resource if found, nil otherwise.
+func (c Client) GetNamespace(ns string) *corev1.Namespace {
+	nsIf, exists, err := c.informers[Namespaces].GetStore().GetByKey(ns)
```

Comment:
One thing I wondered when I read this (could not remember) - is this the list of all namespaces in the K8s cluster or only the ones OSM watches?
Would be great to leave a line here.

Reply:
The implementation at least:
```
monitorNamespaceLabel := map[string]string{constants.OSMKubeResourceMonitorAnnotation: c.meshName}

labelSelector := fields.SelectorFromSet(monitorNamespaceLabel).String()
	option := informers.WithTweakListOptions(func(opt *metav1.ListOptions) {
		opt.LabelSelector = labelSelector
	})
```

I'd assume this works in the way previously described. Let me test it.